canonical-websites/snapcraft.io
# Autoplay video on details pages

Fixes https://github.com/canonicalltd/snap-squad/issues/786
Fixes https://github.com/canonicalltd/snap-squad/issues/787
and implements https://github.com/canonical-websites/snapcraft.io/pull/1461

## QA
- Pull the branch
- `./run`
- Add an appropriate video to a public snap
- Visit the public snap page
- See the video
- Try with youtube, vimeo and asciinema
- Make sure that the videos are OK with and without screenshots
- Make sure the lightbox works when navigating to the video from another screenshot*
- Remove the video text to ensure the video is removed
- /lukewh-test, /test-lukewh and /test-lukewh-2 all have examples


*I've not hacked a way to open the lightbox by clicking on the videos as this seemed like it was overriding default behaviour.
